deluge/ui/tracker_icons.py

```diff
@@ -289,7 +294,14 @@ def parse_html_page(self, page):
         :returns: a Deferred which callbacks a list of available favicons (url, type)
         :rtype: Deferred
         """
-        with open(page) as _file:
+        if chardet:
+            with open(page, 'rb') as _file:
```

Comment:
If it's an HTML page it should not need to be opened in binary mode so can remove `'rb'` option

Reply:
Without `'rb'`, python will try to open it with the default encoding again. It will lead to a `UnicodeDecodeError` error again on Windows.
And tested it also on ubuntu under WSL, which didn't have that problem.
Here is the section of `encoding` param of the `open` function from the docstring:
```
In text mode, if encoding is not specified the encoding used is platform
dependent: locale.getpreferredencoding(False) is called to get the
current locale encoding.
```
When running it from Linux:
```python
>>> locale.getpreferredencoding(False)
'UTF-8'
```

When running it from (my) Windows:
```python
>>>locale.getpreferredencoding(False)
'cp1252'
```
So I'm not sure removing this will help.

Reply:
Okay no problem, I did wonder if I was missing something :smile: